tox.ini

```diff
@@ -33,37 +33,4 @@ commands =
    # specifying the entry point script in `{envbindir}`.
    coverage run -p --module twisted.trial {posargs:towncrier}
    coverage combine -a
-
-[testenv:build]
```

Comment:
Why is this going away?

Reply:
my bad. I thought that we don't need it and that it was already moved to GitHub CI only.

I will revert... 

My hope was that we can use standard tools without the need of extra tools.

Reply:
I wasn't sure where you were going so maybe it isn't needed now?  It was an honest question.

Reply:
This is still need.
The towncrier is built once.
Then the wheel is uploaded as an artifact.
Then the tests will download the artifact and run the tests agains it.

------------

I am not convinced that we need all this dance... but I think is better to refactor this in another PR.